.github/workflows/style.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           pip install black==24.4.2
 
       - name: configure
-        run: cmake -S "cmake/FormatTarget" -Bbuild -DADD_FORMATTARGET=TRUE -DADD_PYTHONFORMATTING=1
+        run: cmake -S "cmake/FormatTarget" -Bbuild -DADD_FORMATTARGET=TRUE
```

Comment:
Then the CI does not check the formating?

Reply:
It still does formatting, as I am duing the ckech directly here via the run command

Reply:
Also see here thats the black output
https://github.com/ikarus-project/ikarus/actions/runs/10178516803/job/28152285734#step:7:17